src/data/polynomial/degree/trailing_degree.lean

```diff
@@ -3,8 +3,21 @@ Copyright (c) 2020 Damiano Testa. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Damiano Testa
 -/
+import tactic
```

Comment:
```suggestion
```

You don't need this. `data.polynomial.degree.basic` imports every tactic you need.